edb/edgeql/codegen.py

```diff
@@ -1225,8 +1225,14 @@ def visit_CreateConcreteProperty(
         keywords = []
         if self.sdlmode and node.declared_overloaded:
             keywords.append('OVERLOADED')
-        if node.is_required:
-            keywords.append('REQUIRED')
+            if node.is_required:
+                keywords.append('REQUIRED')
+        else:
+            if node.is_required is True:
```

Comment:
I always have an itch when I see clauses like this even that there's nothing wrong with them per se. I'd replace with
`if node.is_required is not None: if node.is_required: ... else: ... `

Reply:
The current approach is less nested ```¯\_(ツ)_/¯```